src/client/Microsoft.Identity.Client/Properties/AssemblyInfo.cs

```diff
@@ -4,6 +4,12 @@
 using System;
 using System.Reflection;
 using System.Runtime.InteropServices;
+#if ANDROID_BUILDTIME
```

Comment:
Please use `if ANDROID` as we not hiding anything here.

Reply:
This is probably the reason why the fix isn't working.

Reply:
We tried doing this with #if ANDROID but that did not seem to work. The only thing that did work was adding preserve to each of the classes. We can remove this from the AssemblyInfo